interpreter/text/parser.mly

```diff
@@ -474,7 +474,7 @@ plain_instr :
   | RETURN_CALL var { fun c -> return_call ($2 c func) }
   | RETURN_CALL_REF var { fun c -> return_call_ref ($2 c type_) }
   | CONT_NEW LPAR TYPE var RPAR { fun c -> cont_new ($4 c type_) }
-  | CONT_BIND LPAR TYPE var RPAR { fun c -> cont_bind ($4 c type_) }
+  | CONT_BIND LPAR TYPE var RPAR LPAR TYPE var RPAR { fun c -> cont_bind ($4 c type_) ($8 c type_) }
```

Comment:
See comment in arrange.ml, I think we can omit the `type` here and for cont.new.